test/spec/spectest.cpp

```diff
@@ -612,44 +628,45 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
         return;
       }
       case CommandID::AssertTrap: {
-        const auto &Action = Cmd["action"s];
-        const auto &Text = Cmd["text"s].Get<std::string>();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
-        TrapInvoke(Action, Text, LineNumber);
+        const simdjson::dom::object Action = Cmd["action"].get_object();
+        const std::string_view Text = Cmd["text"];
+        const uint64_t LineNumber = Cmd["line"];
+        TrapInvoke(Action, std::string(Text), LineNumber);
         return;
       }
       case CommandID::AssertExhaustion: {
         // TODO: Add stack overflow mechanism.
         return;
       }
       case CommandID::AssertMalformed: {
-        const auto &ModType = Cmd["module_type"s].Get<std::string>();
-        if (ModType != "binary") {
+        std::string_view ModType = Cmd["module_type"];
+        std::string Binary = "binary";
+        if (Binary.compare(std::string(ModType))) {
           // TODO: Wat is not supported in WasmEdge yet.
           return;
         }
-        const auto Filename = (TestsuiteRoot / Proposal / UnitName /
-                               Cmd["filename"s].Get<std::string>())
-                                  .u8string();
-        const auto &Text = Cmd["text"s].Get<std::string>();
-        TrapLoad(Filename, Text);
+        std::string_view Name = Cmd["filename"];
+        const auto Filename =
+            (TestsuiteRoot / Proposal / UnitName / Name).u8string();
+        Name = Cmd["text"];
+        TrapLoad(Filename, std::string(Name));
         return;
       }
       case CommandID::AssertInvalid: {
-        const auto Filename = (TestsuiteRoot / Proposal / UnitName /
-                               Cmd["filename"s].Get<std::string>())
-                                  .u8string();
-        const auto &Text = Cmd["text"s].Get<std::string>();
-        TrapValidate(Filename, Text);
+        std::string_view Name = Cmd["filename"];
+        const auto Filename =
+            (TestsuiteRoot / Proposal / UnitName / Name).u8string();
+        Name = Cmd["text"];
+        TrapValidate(Filename, std::string(Name));
         return;
       }
       case CommandID::AssertUnlinkable:
       case CommandID::AssertUninstantiable: {
-        const auto Filename = (TestsuiteRoot / Proposal / UnitName /
-                               Cmd["filename"s].Get<std::string>())
-                                  .u8string();
-        const auto &Text = Cmd["text"s].Get<std::string>();
-        TrapInstantiate(Filename, Text);
+        std::string_view Name = Cmd["filename"];
+        const auto Filename =
+            (TestsuiteRoot / Proposal / UnitName / Name).u8string();
+        Name = Cmd["text"];
```

Comment:
Don't use the `Name` for different purposes.

Reply:
Added variable `Text` in https://github.com/WasmEdge/WasmEdge/pull/2659/commits/b903dff5d269911bd3a841257eba8f95fe440630